saleor/graphql/order/mutations/utils.py

```diff
@@ -0,0 +1,19 @@
+from typing import List
+
+from django.utils import timezone
+
+from ....order import OrderStatus
+from ....order.models import Order
+
+
+def invalidate_order_prices(order: Order, *, save: bool) -> List[str]:
+    if order.status not in {OrderStatus.DRAFT, OrderStatus.UNCONFIRMED}:
```

Comment:
We should use `ORDER_EDITABLE_STATUS` from `saleor/saleor/graphql/order/mutations/orders.py`

Reply:
```suggestion
    if order.status not in ORDER_EDITABLE_STATUS:
```